lms/djangoapps/discussion_api/api.py

```diff
@@ -244,6 +248,20 @@ def get_sorted_modules(category):
     }
 
 
+def get_course_topic(request, course_key, topic_id):
+    """
+    Returns details of discussion topic against topic_id
```

Comment:
against => for the given.

Also, should have Arguments, Returns, and Raises sections as usual.